src/main/java/pl/coderslab/charity/entity/User.java

```diff
@@ -0,0 +1,32 @@
+package pl.coderslab.charity.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+
+@RequiredArgsConstructor
+@AllArgsConstructor
+@ToString
+@Getter
+@Setter
+@Entity
+@Table(name = "users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotBlank
+    @Email
+    private String email;
+
+    @NotBlank
+    private String password;
+
+    @NotBlank
+    @Transient
+    private String passwordRepeat;
```

Comment:
Ogólnie no to trochę nie wiem po co zapisywać passwordRepeat. Chyba że adnotacja `@Transient` ci załatwia, że to się nie zapisuje w bazce, ale nadal. Nie dodawałbym tego do encji, tylko zrobił oddzielną klase do przesyłania requestu i potem to mapował do encji w serwisie.

Reply:
zgadza się, @Transient powoduje, że obiekt nie jest mapowany do bazy, postaram się nanieść Twoje uwagi do projektu